src/helpers/scrollUpIfNeeded.ts

```diff
@@ -0,0 +1,34 @@
+import { RefObject } from "react";
+import { VirtuosoHandle } from "react-virtuoso";
+
+type Page = HTMLElement | RefObject<VirtuosoHandle>;
+
+export function scrollUpIfNeeded(activePage: Page | undefined) {
+  if (!activePage) return false;
+
+  if ("querySelector" in activePage) {
+    const scroll =
+      activePage?.querySelector('[data-virtuoso-scroller="true"]') ??
+      activePage
+        ?.querySelector("ion-content")
+        ?.shadowRoot?.querySelector(".inner-scroll");
+
+    if (scroll?.scrollTop) {
+      scroll.scrollTo({ top: 0, behavior: "smooth" });
```

Comment:
Should this be scrollToIndex as below?

Reply:
Seems to work on my local as is, change was merged in from #326 . 

The variable scroll is also of type 'Element' which scrollToIndex does not exist on.

Reply:
Yeah, `scrollToIndex` is a special Virtuoso call. It handles edge cases where Virtuoso changes height of scroll view due to virtual items of unknown height when scrolling up (and other niceties).